### ISSUE TITLE
Cannot get train, valid and test set from the corpus.

### ISSUE THREAD

**Issue by shangqing-liu** [IBM/Project_CodeNet#29]
Hi,

Thanks for this wonderful project.  One question is that when I look through the data, I found that it has not split into train/valid/test. Can you help to split the data accordingly, so we can compare it with the numbers posted in the table.

![image](https://user-images.githubusercontent.com/22492462/124557189-ca30e100-de6b-11eb-9691-d888987fa26b.png)

Thanks and best regards.

**Comment by weidotwisc** [comment]
Hi, 
Thanks a lot for the interest. For the GNN experiments, we conducted a random split of 6/2/2 on the entire dataset (i.e., 60% of the total data goes into training set, 20% of the total dataset goes into test set, and 20% goes into the validation set). 
Thanks! 
Wei

**Comment by shangqing-liu** [comment]
Thanks for the reply, may I ask for the train/valid/test dataset, so we can
compare with the numbers directly rather than split the whole dataset by
the ratio.

**Comment by vthost** [comment]
We'll provide a fixed split soon and will let you know here!

**Comment by shangqing-liu** [comment]
Thanks very much!

**Comment by xieexiaotuzi** [comment]
Thanks for your nice reply.

I would like to ask how to use the split in "Project_CodeNet/model-experiments/gnn-based-experiments/data."? What does the number mean in "Project_CodeNet/model-experiments/gnn-based-experiments/data/C++1000/split/random/train.csv"?

Best

**Comment by PlayeerOne** [comment]
> Thanks for your nice reply.
> 
> I would like to ask how to use the split in "Project_CodeNet/model-experiments/gnn-based-experiments/data."? What does the number mean in "Project_CodeNet/model-experiments/gnn-based-experiments/data/C++1000/split/random/train.csv"?
> 
> Best

I have the same question, what do the numbers mean in the .csv splits ?
